<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeWindow</class>
 <widget class="QDialog" name="ModeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mode</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_calc">
        <property name="text">
         <string>Калькулятор</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_val">
        <property name="text">
         <string>Конвертация единиц</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_prog">
        <property name="text">
         <string>Программист</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
